pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>org.folio</groupId>
   <artifactId>mod-users-bl</artifactId>
   <name>mod-users-bl</name>
-  <version>6.3.0-SNAPSHOT</version>
+  <version>7.0.0-SNAPSHOT</version>
   <licenses>
```

Comment:
Why? Are there any breaking changes for this release as far as the usage / interface of mod-users-bl?

Reply:
I wouldn't as this change is something that only affects system development.

Reply:
Why do you care about that setting? If somebody used it for mod-users-bl, that would be very wrong, since that module is *not* using PostgrsClient. Having said that these versions don't mean much as long as they are increasing in order.

Reply:
> Why do you care about that setting? 

I care because it is provided to consumers of the module and has now been removed.

> If somebody used it for mod-users-bl, that would be very wrong, since that module is _not_ using PostgrsClient.

Yet, due to the design of these modules, it was allowed and now it isn't.

> Having said that these module versions don't mean much as long as they are increasing in order.

Agreed.